### ISSUE TITLE
Add. 17,253, foll. 1–69 | https://bl.vuexistapps.us/manuscript/1046 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1019]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- This entry had two parts in the Hands section. Hand 1 informs of the main script, and Hand 2 refers to Addition 2, indicating that there is an Arabic note on fol. 68b. However, based on the sentence, "The quires are signed with letters from ܝܕ to ܟܐ, and by a later hand from ܝ to ܝܙ" it may be interpreted that we have a third hand that added these numbers. So, in case this is correct, Hand 3 should also be added to the Hands section. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
Add a hand for the signed letters? And what about the note regarding provenance; Wright did not specify if it's the same hand as the hand of the ms

@wlpotter

**Comment by wlpotter** [comment]
Yes, like with #1032 we can add a hand saying that a later hand added quire signatures, but keep the sentence in the phys desc as-is.

We just want the one hand for the later quire signatures, as we don't know if the first set of signatures are the scribe or not. So one hand for the scribe (which also covers addition 1, since it's a colophon); one for the Arabic addition; one for the later quire signatures

**Comment by mikey-tang16** [comment]
hand 3 added
